src/cljs/athens/views.cljs

```diff
@@ -22,10 +23,11 @@
 (def app-wrapper-style
   {:display "grid"
    :grid-template-areas
-   "'left-sidebar main-content secondary-content'
+   "'app-header app-header app-header'
+    'left-sidebar main-content secondary-content'
    'devtool devtool devtool'"
```

Comment:
Might be a good place to leverage garden's vector format

Reply:
```clojure
[["'app-header app-header app-header'"
  "'left-sidebar main-content secondary-content'"
  "'devtool devtool devtool'"]]
```
vs 

```clojure
"'app-header app-header app-header'
 'left-sidebar main-content secondary-content'
 'devtool devtool devtool'"
```

Assuming I'm doing this correctly, given the formatting of the required CSS output, I think the regular string is simpler.

Reply:
Thought we could do something else, but the exepected CSS is a little trickier because it expects a list of strings